server/plugins/DelayUGens.cpp

```diff
@@ -218,38 +218,39 @@ struct DelTapRd : public Unit {
 
 extern "C" {
 
-void SampleRate_Ctor(Unit* unit, int inNumSamples);
-void ControlRate_Ctor(Unit* unit, int inNumSamples);
-void SampleDur_Ctor(Unit* unit, int inNumSamples);
-void ControlDur_Ctor(Unit* unit, int inNumSamples);
-void SubsampleOffset_Ctor(Unit* unit, int inNumSamples);
-void RadiansPerSample_Ctor(Unit* unit, int inNumSamples);
-void NumInputBuses_Ctor(Unit* unit, int inNumSamples);
-void NumOutputBuses_Ctor(Unit* unit, int inNumSamples);
-void NumAudioBuses_Ctor(Unit* unit, int inNumSamples);
-void NumControlBuses_Ctor(Unit* unit, int inNumSamples);
-void NumBuffers_Ctor(Unit* unit, int inNumSamples);
-void NodeID_Ctor(Unit* unit, int inNumSamples);
-void NumRunningSynths_Ctor(Unit* unit, int inNumSamples);
+void SampleRate_Ctor(Unit* unit);
```

Comment:
Why these changes to all the Ctors? Is this just a cleanup since those arguments aren't used? If so, I'd rather see this in a different PR please. Is it necessary for this PR?